README.md

```diff
@@ -1,22 +1,83 @@
-# Zendesk Scripts
+# Zendesk GDPR Cleaner and Manual Scripts
+
+A collection of scripts and pipelines to locate and remove legacy Zendesk tickets and user accounts to meet GDPR
+
+#### Zendesk Notes
+
+* User account status can be either active, soft deleted or hard deleted.
+* Accounts must be soft deleted before being hard deleted - i.e. 2 stages.
+* Soft deleting sets field Active: FALSE
+* Hard deleting removes all PII data but leaves the record and ID in the DB. (as a result, there are >1.2 million records available via the deleted users endpoint).
+* Tickets should be removed BEFORE user accounts are addressed.
+
+#### Zendesk parameters to ensure GDPR compliance
+
+#### Tickets and User Accounts:
+
+* Field "organization_id": null
+* Field "updated_at" > 1 year ago when scripts run
+
+
+#### User Accounts
+
+* Field "role": "end-user"
+* Field "name" [not = zendesk]
+* No. of tickets belonging to user = 0
+
+
 
-A collection of manual tasks (which will be autom8ed) to locate and remove legacy Zendesk tickets and user accounts to meet GDPR
 
 ## Getting Started
 
 * These instructions will get the project up and running on your local machine for development and testing purposes.
-* It is highly recommended that an AWS VM is used for the longer tasks.
-* You will require a Zendesk account which has 'admin' privileges.
+* It is highly recommended that an AWS VM is used for the longer manual scripted tasks.
+* Concourse is the currently preferred automation tool.
+* You will require a Zendesk account which has 'admin' privileges and an associated token.
 * Use of linux 'screen' command is highly recommended
+* Postman has proved very useful for developing and testing queries
+
 
 ### Clone the repo
 
 ```
 git clone git@github.com:alphagov/zendesk-scripts
 ```
 
+### Deploy the pipeline
+
+#### Required 
+
+Download and install correct version of 'fly'
 
-### Installing ruby
+
+#### Login to selected Concourse
+
+```
+fly --target [target-name] login --team-name [team] --concourse-url [concourse-url]
```

Comment:
Can you replace the templates with actual values? Better to give an exact example for the prod multi-tenant concourse, and if you feel it's necessary link people to the `fly login` documentation.

Reply:
Sure, I was concerned about putting real values in there ;-)